lib/modules/manager/buildkite/extract.ts

```diff
@@ -13,7 +13,7 @@ export function extractPackageFile(content: string): PackageFile | null {
     for (const line of lines) {
       // Search each line for plugin names
       const depLineMatch = regEx(
-        /^[\s-]*(?<depName>[^#\s]+)#(?<currentValue>[^:]+)/
+        /^[\s-]*(\?\s)?(?<depName>[^#\s]+)#(?<currentValue>[^:]+)/
```

Comment:
```suggestion
        /^\s*(?:-\s+(?:\?\s+)?)?(?<depName>[^#\s]+)#(?<currentValue>[^:]+)/
```
I'm not sure, but is the dash optional for those plugins?

Reply:
It is optional, and this suggestion works perfectly!